### DISCUSSION TITLE
Add icon adornment to input

### DISCUSSION THREAD

**Discussion by olee** [saadeghi/daisyui#2238] [Q&A] [answered]
Hey, I couldn't find anything in the docs related on how to add icons to inputs like [this](https://mui.com/material-ui/react-text-field/#icons) for example.
Is there any standard solution on how to achieve this?

**Comment by orbiteleven** [comment]
@olee did you ever figure this out?

**Comment by saadeghi** [comment, marked as answer]
You can do something like this:  
https://play.tailwindcss.com/17Qzpw6JKv
```html
<label class="input input-bordered inline-flex items-center gap-2">
  🔒
  <input class="h-full" value="Email"/>
</label>
```
